pkg/acsengine/templates.go

```diff
@@ -0,0 +1,1707 @@
+// Code generated by go-bindata.
```

Comment:
You might be working off of an old branch as `pkg/acsengine/templates.go` is not checked into source control. Please delete it.